Documentation/contributing.md

```diff
@@ -6,6 +6,8 @@ Consistent code conventions are important for several reasons:
 * To make it easy to debug the code, with both a system call tracer and GNU debuggers. It should be easy to set breakpoints, view locals, and display and view data structures.
 * To attempt to improve code quality through consistency, and requiring patterns that are less likely to result in bugs either initially, or after code modification.
```

Comment:
and requiring patterns **that** are less likely

Reply:
It's not an error.